networking/hardware_networks/about-sriov.adoc

```diff
@@ -5,12 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can use Single Root I/O Virtualization (SR-IOV) network devices with additional networks on your {product-title} cluster for high performance applications.
+Developed by the PCI-SIG (PCI Special Interest Group), the Single Root I/O Virtualization (SR-IOV) specification is a standard for a type of PCI device assignment that can share a single device with multiple pods.
 
-[id="about-using-sr-iov-hardware"]
-== About SR-IOV hardware on {product-title}
+SR-IOV enables you to segment a compliant network device, recognized on the host node as a Physical Function (PF), into multiple Virtual Functions (VFs).
```

Comment:
@zshi-redhat, I don't know if any of this is actually correct; The goal is to expand upon what SR-IOV even is.

Reply:
yes, it's correct

Reply:
@zshi-redhat, awesome!